src/services/copyConfig/consent/consentConfiguration.test.js

```diff
@@ -77,7 +84,7 @@ describe('ConsentConfiguration test suite', () => {
     const mockedResponse = getResponseWithContext(expectedGigyaResponseInvalidAPI, 'consent_consentStatement_get', apiKey)
     axios.mockResolvedValueOnce({ data: mockedResponse })
 
-    const responses = await consentConfiguration.copy(apiKey, { dataCenter }, consentOptions.getOptions())
+    const responses = await consentConfiguration.copy(apiKey, { dataCenter }, optionsConsent)
```

Comment:
Use the class ConsentOptions instead of an arbitrary object.
The branches array should be in ConsentOptions class